pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+    This function uses Fisher's definition of kurtosis (kurtosis of normal
+    == 0.0) without bias.
+
+    Returns
+    -------
+    same type as input
```

Comment:
It's a somehow arbitrary comment, but to me it'd look better something like:
`Series or DataFrame (same as the input) : some description`

Or if the `%(name)s` above `Series` or `DataFrame` depending on in which method is being used? Then we can use it also here.